### PR TITLE
decrease announcement padding; refactor

### PR DESCRIPTION
**Pull request by nyiyui**
- use `async` over `Promise`
- use `fetch` over `$.getJSON`
- etc

### CODE REVIEW

**Comment by nyiyui** [comment]
> 

Ah,

**Comment by nyiyui** [comment]
> Technically I didn't put it in the html because the qr code loads with the announcements and if the announcements fail itll just be a floating msg

Changed so it's hidden by default (563c5a3964684aa19198dda05830bce831e2d47c).

File: js/information.js [hunk 1]
```diff
@@ -343,47 +343,49 @@ function updateSchedule() {
...
-  })
+  getWeather()
     .then(() => {
       // Sets weather data once the promise is resolved
-      $("#w-icon").show();
       $("#w-icon").attr("src", `img/weathericons/${weather.WeatherIcon}.svg`);
-      $("#temp").text(`${weather.Temperature.Metric.Value.toFixed(1)}°C`);
+      $("#w-icon").show();
+      $("#temp").text(
+        `${Math.round(weather.Temperature.Metric.Value.toFixed(1))}°C`
```
Comment (rcshim04) [inline js/information.js:332]: Remove .toFixed(1), and where are you setting the value for weather?

Comment (nyiyui) [reply js/information.js:332]: Should be fixed now. Thanks!

**Pull request merged**